src/app/pages/about/about.component.html

```diff
@@ -83,6 +97,13 @@ <h1 class="vf-text vf-text-heading--4">Eligibility Criteria</h1>
               Contain at least some samples derived from healthy human donors
             </li>
             <li>Be derived from a recognised single cell technology.</li>
+            <li>
+              Be available in raw fastq format and/or as a gene expression matrix
+              (raw counts or processed counts). Multiplexed data must be demultiplexed
+              prior to upload of raw fastq files. If the fastq data is not able to be
+              demultiplexed, it is not eligible. In this case, a gene expression matrix
+              can be uploaded instead, if available.
```

Comment:
Same change as the suggestions above needed here - it's a bit painful to copy / paste within a PR so I won't include again! :)